### PR TITLE
[DM-3154] Grid filter text not getting clear

### PR DESCRIPTION
**Pull request by bozhidardedov**

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-10-18-0-227-10-19-5-13-10-18-503-44-grid-filter-text-not-getting-clear.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3154
+version: 10.18.0.227,10.19.5.13,10.18.503.44
+---
+In data grid columns using the simple FilteringFormRendererComponent to display filter input, the input value was not cleared even though the filter itself was reset. This is now fixed and input value is cleared when filter is reset.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-10-18-0-227-10-19-5-13-10-18-503-44-grid-filter-text-not-getting-clear.md:17]:
```suggestion
In data grid columns using the simple FilteringFormRendererComponent to display filter input, the input value was not cleared even though the filter itself was reset. This is now fixed and input value is cleared when filter is reset.

When using the simple `FilteringFormRendererComponent` in data grid columns to display filter inputs, the input values were not cleared when the filter was reset. This issue has been fixed - now when the filter is reset, the input values are properly cleared.
```

File: content/change-logs/device-management/ui-c8y-10-18-0-227-10-19-5-13-10-18-503-44-grid-filter-text-not-getting-clear.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Grid filter text not getting clear
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-10-18-0-227-10-19-5-13-10-18-503-44-grid-filter-text-not-getting-clear.md:3]:
```suggestion
title: Grid filter text not cleared on filter reset
```

**Comment by BeateRixen** [comment]
@bozhidardedov Do you want to change the file name to match the changes in the title? It is not mandatory to do so, depending how much it differs from the original file name. This is actually something we are still unsure about :-).

**Review by BeateRixen — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Comment by bozhidardedov** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|release/y2024|[](https://github.com/SoftwareAG/c8y-docs/pull/1743)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**